server/public/model/config.go

```diff
@@ -1002,6 +1003,10 @@ func (s *MetricsSettings) SetDefaults() {
 	if s.BlockProfileRate == nil {
 		s.BlockProfileRate = NewInt(0)
 	}
+
+	if s.EnableClientMetrics == nil {
+		s.EnableClientMetrics = NewBool(true)
```

Comment:
This should be `false` if we are considering this feature to be "default off". cc @nab-77

Reply:
I think I said default off on my other PR, but since metrics are default off and because this is behind a feature flag for now, I figured making it default on would make sense. While we still have the feature flag, it'll only be enabled by people who mean to turn it on, and after that, I'd think that anyone who wants to monitor server performance will also care about at least basic client performance as well. Only 1/5 on that though

Reply:
Gotcha, I just wanted to verify with @nab-77 otherwise it makes sense to have this default on.